src/helpers/asce.js

```diff
@@ -39,12 +39,12 @@ const faAsce = (Ss, tp) => {
   return F[tp][5];
 };
 
-const asce = async (location, soilType) => {
+const asce = async (lat, long, location, soilType) => {

```

Comment:
what is the difference between lat, long and location?

Reply:
Location has ID as its value (obtained from getLocation, when the component is mounted). We store this value in a global state, The purpose is to use "location" only for searching on zar's models(by id). 
As a result, **the request to DB to obtain latitude and longitude do not happen** (we pass latitude and longitude by parameters, wich are already stored in the global state)